options.json

```diff
@@ -31,10 +33,11 @@
    "text_color": "#FFFFFF", 
    "trackerserver_authkey": "", 
    "trackerserver_url": "http://isaacitemtracker.com:8080", 
-   "twitch_name": "dea1h", 
-   "width": 1315, 
+   "transparent_mode": false, 
+   "twitch_name": "", 
+   "width": 1078, 
    "word_wrap": false, 
    "write_to_server": false, 
-   "x_position": 186, 
-   "y_position": 926
+   "x_position": 573, 
+   "y_position": 886
 }
```

Comment:
General comment on this file @Hyphen-ated 

This should probably be included in the `.gitignore` and removed from the repo. These are runtime settings for the development build. `options_default.json` is the file that matters.

Reply:
I vaguely remember there having been a reason I didn't gitignore this file, but I have no idea now what that reason might have been. Maybe it no longer applies